sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi_simple.c

```diff
@@ -288,50 +288,8 @@ static inline void finite_difference(float output[3], float new[3], float old[3]
  * @param att_err quaternion attitude error
  * @param rate_control rate control enabled, otherwise attitude control
  */
-static inline void stabilization_indi_calc_cmd(int32_t indi_commands[], struct Int32Quat *att_err, bool rate_control)
+void stabilization_indi_calc_cmd(int32_t indi_commands[])
```

Comment:
Something seems wrong here. `indi_commands` is the output vector, so their is no more setpoint input to this function. It should be the same `rate_ref` than full indi, from which the acceleration setpoint is computed.
As a side note, the same function in full indi code is directly writing the commands the the `actuators_pprz` array. Maybe it is better to pass the output array as an argument like it is done here.

Reply:
Would you recomend having the same function in both indi codes with the same inputs?